index.html

```diff
@@ -3,17 +3,91 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Nice job using the viewport meta tag. It can be easy to forget this because it can look like it's not making a difference until you actually view the site on a mobile device.